examples/gno.land/r/gov/dao/v2/prop4_filetest.gno

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"gno.land/p/demo/dao"
+	"gno.land/r/gov/dao/bridge"
+	govdaov2 "gno.land/r/gov/dao/v2"
+	"gno.land/r/sys/params"
+)
+
+func init() {
+	mExec := params.NewStringPropExecutor("prop1.string", "value1")
```

Comment:
Maybe this prop test should be in `sys/params`?

Reply:
Right now, we can't verify if the setter worked because there is only std.Set, not std.Get. 

Therefore, I believe the only thing we can test is the govdao flow involving a proposal. 

I can add a unit test that sets something from sys/params to ensure there is no panic.

Reply:
fbf4fdb58